_posts/en/posts/2016-01-25-segwit-benefits.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+name: segwit-benefits
+id: en-segwit-benefits
+title: Segregated Witness Features
+permalink: /en/2016/01/25/segwit-benefits
+version: 1
+---
+1. toc
+{:toc}
+
+The Segregated Witness soft-fork (segwit) includes a wide range of
+features, many of which are highly technical. This page summarises
+some of the benefits of those features that go beyond simply increasing
+the capacity.
+
+## Malleability Fixes
+
+Bitcoin transactions are identified by a 64-digit hexidecimal hash,
+which is based on both the previous coins being spent, and on who
+will be able to spend the results of the transaction. Unfortunately,
+the way the hash is calculated allows anyone to make small
+modifications to the transaction that will not change its meaning,
+but will change the hash. So you could submit a transaction with hash
+ef74c1cbf0003fc4e96a87a59838f7dd3da488d9d83fec3f270b0d3d7c2bc309 to the
+network, but instead find that the network confirms that transaction under
+the hash 683f45578328242a06bc5c54acbcbe6e70a5435b4561fc8b0570a59ab09f8bfa
+instead.
+
+Segwit prevents malleability by allowing bitcoin users to move the
+malleable parts of the transaction into the transaction witness, and
+segregating that witness so that changes to the witness does not affect
+calculation of the transaction's hash.
+
+Previous attempts to reduce malleability have included BIP66 (strict
+DER encoding for signatures) and BIP62 ("dealing with malleability",
+withdrawn).
+
+#### Who benefits?
+
+Avoiding transaction malleability is helpful for anyone monitoring a
+transaction to determine when it is confirmed. Transaction malleability
+also causes problems for spending unconfirmed transactions, or when
+building smart contracts between mutually untrusting users. The lightning
+network requires a fix for third-party transaction malleability before
```

Comment:
Personally I will not start working on Lightning before segwit and CSV passed, lightning is too boring with it.

http://diyhpl.us/wiki/transcripts/scalingbitcoin/hong-kong/overview-of-bips-necessary-for-lightning/